apps/ex_wire/lib/ex_wire/p2p/server.ex

```diff
@@ -147,27 +158,24 @@ defmodule ExWire.P2P.Server do
     {:reply, :ok, new_state}
   end
 
-  def handle_info({:connect, opts}, %{peer: peer}) do
-    {:ok, socket} = TCP.connect(peer.host, peer.port)
+  def handle_info(:connect, state = %{peer: peer}) do
+    {:ok, socket0} = TCP.connect(peer.host, peer.port)
 
     :ok =
       Logger.debug(fn ->
         "[Network] [#{peer}] Established outbound connection with #{peer.host_name}."
       end)
 
-    new_conn =
-      socket
-      |> Manager.new_outbound_connection(peer)
-      |> Map.put(:subscribers, Map.get(opts, :subscribers, []))
+    state0 = Manager.new_outbound_connection(%{state | socket: socket0})
```

Comment:
Would prefer not numbering variables or at least making them sequential. `state0` seems like it would be the state before `state`.

Reply:
Indeed, this is an error.